compiler/pipes/collect-main-edges.cpp

```diff
@@ -37,6 +39,8 @@ SwitchKind get_switch_kind(VertexAdaptor<op_switch> s) {
     if (VertexUtil::is_const_int(val)) {
       num_const_int_cases++;
     } else if (auto as_string = val.try_as<op_string>()) {
+      // In case of op_switch node that was generated from op_match
+      num_const_total_string_cases++;
```

Comment:
I don't understand the name of this variable. Probably, it should be "num_string_cases"?